onnxruntime/python/tools/tensorrt/perf/perf_utils.py

```diff
@@ -58,6 +58,19 @@
 extended = 'extended'
 enable_all = 'all'
 
+# default TRT EP options
+enable_cache_key = "trt_engine_cache_enable"
+max_workspace_key = "trt_max_workspace_size"
+default_trt_ep_options = {max_workspace_key: "4294967296", enable_cache_key: "True"}
```

Comment:
Because this is only used for the benchmark default args, we can just pass in {"trt_engine_cache_enable": "4294967296", "trt_max_workspace_size": "True"} to avoid too references.